Api/docs/postman/SalesAgility.postman_collection.json

```diff
@@ -0,0 +1,934 @@
+{
```

Comment:
why this json for? where do we use it?

Reply:
You can import example endpoints into your postman. Try it :)